quinn-proto/src/shared.rs

```diff
@@ -257,33 +273,81 @@ pub struct ServerConfig<S>
 where
     S: crypto::Session,
 {
-    /// Transport configuration to use for incoming connections
+    /// TLS configuration used for incoming connections.
     pub transport: Arc<TransportConfig>,
+    /// Mutable TLS configuration used for incoming connections.
+    pub crypto: S::ServerConfig,
```

Comment:
They are used by the benchmarks, interop, tests, that do not belong to the crate, and can't reference those fields. Since getters are not allowed we have to expose those fields.

Reply:
Can you provide specific examples of cases outside quinn-proto where either of these fields are read from?

Reply:
quinn\src\builders.rs:132:9
quinn\src\builders.rs:142:9
self.config.crypto.set_protocols(protocols);

Reply:
I see. Maybe we can refactor `ServerConfigBuilder` and `ClientConfigBuilder` to render those accesses unnecessary, but it looks like that may not be trivial. In that light I'm content to leave them public for now.